recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -45,6 +45,9 @@ KBUILD_DEFCONFIG_x64 = "nati_x86_64_defconfig"
 KBUILD_DEFCONFIG_armv7a = "nati_zynq_defconfig"
 KCONFIG_MODE="--alldefconfig"
 
+# autoload if installed
+KERNEL_MODULE_AUTOLOAD += "ehci-hcd g_ether xilinx-xadcps"
+
```

Comment:
>xilinx-xadcps

Are some of these modules ARM only? Is that going to be a problem if you define them here?

Reply:
Won't be a problem if the modules aren't built for x64. Only when a module's ipk is built will this setting make a difference.